phys2bids/tests/test_txt.py

```diff
@@ -1,5 +1,4 @@
 import math
```

Comment:
```suggestion
import math

```

Need the blank line; `pytest` is third-party.

Reply:
Need this blank line. `pytest` is third-party!